ddbToFirehoseLambda/index.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
To be consistent, this should be in the `persistence` package. Similar to how `ddbToES` is also in the `persistence` package.

Reply:
updated PR for persistence and this PR to move this code. This PR will fail until persistence is merged and published to pickup the exports of the function.